library/Hooks-windows.cpp

```diff
@@ -836,6 +820,9 @@ bool FirstCall()
     _SDL_EnableUNICODE(1);
 
     fprintf(stderr,"Initized HOOKS!\n");
-    inited = true;
-    return 1;
+}
+
+void InitSDLPointers()
+{
+    std::call_once(inited, [](){ FirstCall(); });
```

Comment:
Is there a reason for the lambda instead of just `call_once(inited, FirstCall)`? From https://en.cppreference.com/w/cpp/thread/call_once, the second parameter is a Callable, and I'd expect a function pointer to qualify for that, but I haven't tried it myself, so I could be wrong.

Reply:
Since I can't compile on windows and since I have used a lambda before in this kind of case I just figured it would be easier, I have no strong reason for using one over the other.